cache_run_c_code = 'Test does not build: Traceback (most recent call last):\n  File "/media/psf/iCloud/Uni/Masterthesis/ss20-ma-kaywallasshek-ml-mptcp/ns3-mptcp/.waf-1.7.13-5a064c2686fe54de4e11018d22148cfc/waflib/Tools/c_config.py", line 459, in run_c_code\n    bld.compile()\n  File "/media/psf/iCloud/Uni/Masterthesis/ss20-ma-kaywallasshek-ml-mptcp/ns3-mptcp/.waf-1.7.13-5a064c2686fe54de4e11018d22148cfc/waflib/Build.py", line 188, in compile\n    raise Errors.BuildError(self.producer.error)\nBuildError: Build failed\n -> task in \'testprog\' failed (exit status 1): \n\t{task 140283694553488: cxxprogram test.cpp.1.o -> testprog}\n[\'/usr/bin/g++\', \'-Wl,-Bsymbolic-functions\', \'-Wl,-z,relro\', \'test.cpp.1.o\', \'-o\', \'/media/psf/iCloud/Uni/Masterthesis/ss20-ma-kaywallasshek-ml-mptcp/ns3-mptcp/build/.conf_check_3b2479b91ea1f8aedc2bbc867fa4858e/testbuild/testprog\', \'-Wl,-Bstatic\', \'-Wl,-Bdynamic\', \'-L/usr/lib\', \'-lpython2.7\']\n'
